test/dash/dash_parser_manifest_unit.js

```diff
@@ -343,16 +343,9 @@ describe('DashParser Manifest', () => {
           '        <SegmentTemplate media="1.mp4" duration="1" />',
           '      </Representation>',
           '    </AdaptationSet>',
-          '    <AdaptationSet mimeType="video/mp4" lang="fr" group="1">',
-          '      <Accessibility schemeIdUri="urn:scte:dash:cc:cea-608:2015"',
-          '         value="CC1=lang:fr;CC3=lang:zh"/>',
```

Comment:
After looking at the DASH specification, I've determined values like these are actually invalid, so we shouldn't have been handling them correctly anyways. I've updated the `dash_parser.js` to parse CEA related stuff properly and updated the unit tests here.